docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -417,27 +417,32 @@ image:images/ccs/ccs-min-roundtrip-client-response.svg[]
 [[ccs-supported-configurations]]
 === Supported {ccs} configurations
 
-Elastic only supports searches from a local cluster to a remote cluster running:
+In 8.0+, Elastic supports searches from a local cluster to a remote cluster
+running:
 
 * The previous minor version.
 * The same version.
-* A newer version. This version must also be compatible with the local cluster
-as outlined in the following matrix.
-+
-[%collapsible]
+* A newer minor version in the same major version.
+
+Elastic also supports searches from a local cluster running the last minor
+version of a major version to a remote cluster running any minor version in the
+following major version. For example, a local 7.17 cluster can search any
+remote 8.x cluster.
+
 [[ccs-version-compatibility]]
-.Version compatibility matrix
-====
-include::{es-repo-dir}/modules/remote-clusters-shared.asciidoc[tag=remote-cluster-compatibility-matrix]
-====
+include::{es-repo-dir}/search/search-your-data/ccs-version-compat-matrix.asciidoc[]
 
 IMPORTANT: For the <<eql-search-api,EQL search API>>, the local and remote
 clusters must use the same {es} version.
 
-For example, a local 8.0 cluster can search a remote {prev-major-last}, 8.0, or
-8.1 cluster. However, a search from a local 8.0 cluster to a remote 7.16 or 6.8
+For example, a local 8.0 cluster can search a remote 7.17 or any remote 8.x
+cluster. However, a search from a local 8.0 cluster to a remote 7.16 or 6.8
 cluster is not supported.
 
+Only features that exist across all searched clusters are supported. Using a
+recent feature with a remote cluster where the feature is not supported will
+result in undefined behavior.
```

Comment:
I wonder if there is room for a special mention that for "forward" compatibility, we only support un-deprecated functionality when talking to the next major. The reasoning being that if we deprecate something e.g. in 8.x, as far as I understand the new make-it-minor policy right, we can remove in 9.0 if its been deprecated long enough (i.e. the other time-based criteria are met). That means the feature is still in 8.last (deprecated, will emmit warnings) but we won't support it on 9.0.

Reply:
That's a good point. I pushed https://github.com/elastic/elasticsearch/pull/84055/commits/88c62c502c69156f9672a92c6f59d0e9c8bf528a to remove "recent" from the second sentence. With that change, this paragraph will now apply to BWC and FWC remote clusters.